### ISSUE TITLE
switch action sets - always rerun last command 

### ISSUE THREAD

**Issue by charlierudolph** [Originate/tertestrial-server#59]
Not certain about this but I think it makes. Here is my example

```yml
# tertestrial.yml
actions:
  test:
    - match:
        filename: '\_spec.coffee$'
      command: 'mycha --reporter spec {{filename}}'

  lint:
    - match:
        filename: '\.coffee$'
      command: 'coffeelint {{filename}}'
```

* start tertestrial (uses action set `test`)
* trigger test on `app.coffee` (meaning to lint)
  * It errors with `Error: no matching action found for {"filename":"app.coffee"}`.
* cycle action sets to `lint` 
  * nothing happens. (I think this should repeat the last command even though it failed)
* trigger test 
  * works
* cycle action sets to `test`
  * errors with `Error: cannot find a template for '[object Object]'` (This seems wrong)

Basically it appears the last command is not updated if no matching action is found. I believe this is wrong and we should store the last command no matter what. Here is another example which makes it feel wrong

* start tertestrial (uses action set `test`
* trigger test on spec file
  * works
* trigger test on source file (meaning to lint)
  * errors with no matching action
* cycle action set to `lint`
  * lints the spec file instead of the source file
